test/integration/transactions/payment.ts

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash'
+
+import { Payment } from 'xrpl-local'
+
+import serverUrl from '../serverUrl'
+import { setupClient, suiteClientSetup, teardownClient } from '../setup'
+import { generateFundedWallet, testTransaction } from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('Payment', function () {
+  this.timeout(TIMEOUT)
+
+  before(suiteClientSetup)
+  beforeEach(_.partial(setupClient, serverUrl))
+  afterEach(teardownClient)
+
+  it('base', async function () {
+    const wallet2 = await generateFundedWallet(this.client)
+    const tx: Payment = {
+      TransactionType: 'Payment',
+      Account: this.wallet.getClassicAddress(),
+      Destination: wallet2.getClassicAddress(),
+      // 2 times the amount needed for a new account (20 XRP)
+      Amount: '1000',
```

Comment:
Is this comment still relevant? (Not sure what it means if so)

Reply:
whoops, copy-paste error